databuilder/query_model.py

```diff
@@ -428,7 +428,7 @@ def get_input_domains(node):
 
 
 # We use an arbitrary unique object to represent the patient domain
-PATIENT_DOMAIN = frozenset([object()])
+PATIENT_DOMAIN = frozenset(["PatientDomain"])
```

Comment:
I'm not dead set on this since it allows in principle for bugs where we accidentally construct another set that just contains `"PatientDomain"` which would be considered equal to this; but that seems pretty unlikely and this was helpful with early debugging when I was still coming to grips with the way it worked.